### PR TITLE
Add CVE-2019-6486 to bugs.md

### PR DESCRIPTION
**Pull request by FiloSottile**

### CODE REVIEW

**Comment by alex** [comment]
Is the actual test case for this in OSS wycheproof yet?

**Comment by alex** [comment]
Also I think this should be added to https://github.com/google/wycheproof/blob/master/doc/bugs.md

**Comment by FiloSottile** [comment]
> Is the actual test case for this in OSS wycheproof yet?

Not yet!

> Also I think this should be added to https://github.com/google/wycheproof/blob/master/doc/bugs.md

Ah, thanks, will amend.

File: doc/bugs.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ See [list of issues](issues.md) for details.
 |           Summary                |            Credits         |         CVE       |       Upstream Acknowledgement          | Tests                                  |
 |:---------------------------:     |:--------------------------:|:-----------------:|:--------------------------------------: |:---------------------------------------:
 | GCM wrapped around counter       | Quan Nguyen                |  N/A              |   goo.gl/OdhZcY  |
+| P-384 and P-521 ScalarMult DoS   | Daniel Bleichenbacher      |  CVE-2019-6486    |  [golang/go#29903](https://github.com/golang/go/issues/29903) | To be released
```
Comment (thaidn) [inline doc/bugs.md]: I think this bug was discovered by Harris Baskaran who ran Wycheproof against Golang.

**Pull request merged**